### PR TITLE
hotfix

### PR DESCRIPTION
**Pull request by joerocca**

### CODE REVIEW

File: js/index-min.js [hunk 1]
```diff
@@ -1,7 +1 @@
...
+var $=jQuery;$(document).ready(function(){$(window).scroll(function(){($(this).scrollTop()>60||$(".menu").hasClass("open"))&&$(".menu").css("background","#697061")});var o=$("html, body");$("a.scroll-tag").click(function(){if($(".menu.open").removeClass("open").addClass("closed"),"#"==$(this).attr("href")[0])return event.preventDefault(),o.animate({scrollTop:$('[name="'+$.attr(this,"href").substr(1)+'"]').offset().top-80},450),!1}),$("#copyright").on("click",function(){$(".image-block img").each((o,s)=>{$(s).attr("src",function(o){var s=$(o).attr("src").split(".");return s[0]=s[0]+"-refined",s.join(".")}(s))})});var s=0;$("#gocrazy").on("click",function(){5==s&&($(".image-block img").each((o,s)=>{$(s).attr("src","images/gocrazy.gif")}),$("#gocrazy").append('<audio loop autoplay><source src="sounds/gocrazy.mp3" type="audio/mpeg"></audio>')),s++}),$(".menu").on("click",function(){$(".menu").hasClass("closed")?$(".menu.closed").removeClass("closed").addClass("open"):$(".menu.open").removeClass("open").addClass("closed")}),$("#rsvp-form").on("submit",function(){$(this).css("opacity",0),$("#form-submitted").css("opacity",1)})});
```
Comment (gabepages) [inline js/index-min.js]: 1. Can you please change the `s` & `o`'s to readable variable names? I would prefer ease of reading over how slick the code looks. 

2. Can we leave the `refineImage(element)` part. I think it makes it a little easier to understand what we are changing the image `src` too. 

3. Can you change `5 == s` to `s == 5`, just OCD lol.

**Review by gabepages — CHANGES_REQUESTED** [review]
Love the egg, lets clean up this JS a little though and make it easier to read for future use.

**Comment by gabepages** [comment]
Also, please resolve merge conflicts.

File: js/index-min.js (hunk 1 above)
Comment (joerocca) [reply js/index-min.js]: whatever man this is bs

**Comment by joerocca** [comment]
GTG

**Review by gabepages — APPROVED** [review]

**Pull request merged**